CONTRIBUTING.md

```diff
@@ -207,6 +249,8 @@ Make sure the icon is added in alphabetical order. If you're in doubt, you can a
 
 #### Optional Data
 
+If you needed to add a modifier to the icon's file name above to resolve a clash with the name of an existing icon then you will also need to add a `slug` entry to the icon's JSON and set the value to your chosen file name.
```

Comment:
Because you use "also" in this sentence, I feel like it should come after the list of optional fields. Also I think the sentence can be improved a bit, feel free to improve upon this

```suggestion
If you used a modifier for the icon's file name to resolve a clash with the name of an existing icon then you also need to set the value of `slug` to your chosen file name to the icon's data in `_data/simple-icons.json`.
```

I'm also fine with leaving it at the beginning of this subsection, in which case "also" should be removed. However, given that the custom slug should be very uncommon I think we should put it after the data that is more common.

Reply:
My thinking on putting it first was that, uncommon as it may be, in cases when it is needed it is arguably more important than `guidelines` or `license` as the exclusion of `slug` will cause the linting to fail while the exclusion of either or both of the others won't. That an it also comes before the other 2 in the JSON! 🙂

Changed the wording a bit, making it a bit more succinct and added it to the other 2 bullet points.

Reply:
This seems like a good compromise :+1: